docs/whats-new/zowe-v3-frequently-asked-questions.md

```diff
@@ -0,0 +1,123 @@
+
+3. How does client certificate authentication work in Zowe V3?
+
+   Northbound authentication accepts client certificates. The API Mediation Layer then transforms the client certificate to another method of authentication (such as a JWT token, SAF IDT, or passticket) and this new method is then accepted by southbound services. Currently, we do not plan to support authentication with client certificates to z/OSMF.
+
+4. Will you identify deprecated functionality such as passtickets? **[where will this functionality be identified? are passtickets really deprecated? first FAQ says no.]**
+
+   We plan to identify deprecated functionality. We will announce how this functionality will be identified in the near future.
+
+5. How do you intend to work with the the bypass scheme?
+
+   The bypass scheme remains and you can still claim conformance for services that only provide public endpoints that do not require authentication.
+
+6. Do the dependency changes mean that I need Java 17 to run Zowe V3?
+
+   Yes. Java 17 is required to run the API Mediation Layer in Zowe V3.
+
+### Zowe Explorer for Visual Studio Code
+
+1. Will the functionality to convert Zowe V1 profiles to team configuration remain available for Zowe V3?
+
+Yes. However, users will not be able to use Zowe V1 profiles to connect to services on the mainframe.
+
+2. Will the APIs that Zowe CLI provides for extenders remain the same?
+
+In broad terms, yes. However, some of the functionaity that was available in Zowe V2 has been changed (or removed). The changes are listed in these [CLI](https://github.com/zowe/zowe-cli/issues/1694) and [Imperative](https://github.com/zowe/imperative/issues/970) lists of breaking changes. **[use box docs]**
+
+### Zowe Application Framework, ZSS
+
+1. Is it possible to run 32-bit plug-ins at the same time as 64-bit plug-ins with ZSS?
+
+   No. ZSS runs in either 32-bit or 64-bit mode, which means it can accommodate only one type of plug-in.
+
+2. Are **[what kind of?]** libraries updated to the current latest versions?
+
+In Zowe V3, the Application Framework uses Angular 16 and Webpack 5, which are the latest current releases.**[are there other libraries that aren't up-to-date, and if so, should we list them here?]**
+
+3. What React versions are supported by Zowe Desktop?
+
+It is possible for Desktop extensions to use different versions of React. While the Desktop is not built on React, it is still compatible with extensions that import React.
+
+4. Is the ZEN application running on Zowe Desktop or on z/OS?
+
+ZEN runs on a laptop/desktop and connects to z/OS via standard methods.
+
+5. Is it possible to validate and change the `zowe.yaml` file and job definition within ZEN?
+
+Yes.
+
+6. What happens if in ZEN you click on the SMP/E? **[what is smp/e here? is it a button? a drop-down option? how do users get to smp/e?]**
```

Comment:
@balhar-jakub - please respond

Reply:
SMP/E is an option that you can select within the UI, probably a button but @1000TurquoisePogs know more details.

Reply:
The Zowe Server Install Wizard supports installation of Zowe servers via the PAX or SMPE methods. For more information refer to https://github.com/zowe/docs-site/pull/3663 documentation

Reply:
This question and response does not provide the reader with any valuable information. The response is self-evident. I suggest the question either be removed or, if there is any additional information about how the wizard assists with the other install methods, for instance, that one particular method is the easist to configure for testing puposes (i.e. convenience build) then the question should be something more like, "What is the most straight-forward method of servier side installation made available through the Zowe Server Install Wizard?". Another valuable question could be, "Are the fields in which the user needs to provide inputs different according to the installation option I choose in the Zowe Server Install Wizard?" This answer would be useful as it explains that the various installation methods present different fields which the user is asked to populate. Isn't it in fact true that some option, perhaps via SMP/E installs the runtime automatically?

Reply:
I removed the question.